CriticalMassTests/TwitterManagerTests.swift

```diff
@@ -18,25 +18,31 @@ class TwitterManagerTests: XCTestCase {
     func testUpdateMessagesCallback() {
```

Comment:
Maybe worth extending the existing tests or adding new tests for the new content states like loading

Reply:
Yes. Think so too we should have some tests

Reply:
Added some tests